src/Scene/NodeProcess.js

```diff
@@ -99,6 +100,7 @@ NodeProcess.prototype.subdivideNode = function subdivideNode(node, camera, param
                 // update Imagery wmts
                 for (const layer of colorLayers) {
                     if (layer.tileInsideLimit(child, layer)) {
+                        TiledImageTools.computeTileWMTSCoordinates(child, layer);
```

Comment:
This is done here because we need to know the WMTS coordinates to know if we require new texture or not